samples/highcharts/website/histogram-symmetrical-distribution/demo.html

```diff
@@ -0,0 +1,211 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/highcharts-more.js"></script>
+<script src="https://code.highcharts.com/modules/exporting.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<div id="container" style="width:350px;height:250px;margin:1.5em 1em;"></div>
+
+<script>
```

Comment:
Usually we put JS code in `demo.js`... Is there a particular reason for putting it in `demo.html`?

Reply:
No, I don't know why it is in the HTML file...I can move it to `demo.js`

Reply:
All the demos are used in the blog so I'll move them to /blog.